packages/web3-core/src/web3_config.ts

```diff
@@ -133,6 +175,11 @@ export abstract class Web3Config
 		this._config.defaultBlock = val;
 	}
 
+	/**
+	 * The `transactionBlockTimeout is used over socket-based connections. This option defines the amount of new blocks it should wait until the first confirmation happens, otherwise the PromiEvent rejects with a timeout error.
```

Comment:
```suggestion
	 * The `transactionBlockTimeout` is used over socket-based connections. This option defines the amount of new blocks it should wait until the first confirmation happens, otherwise the PromiEvent rejects with a timeout error.
```